CONTRIBUTING.md

```diff
@@ -51,7 +51,20 @@ The format is a gzipped JSON `genesis.json` file, with either:
 - a `stateHash` attribute: to omit a large state (e.g. for networks with a re-genesis or migration history).
   Nodes can load the genesis block header, and state-sync to complete the node initialization.
 
-### 4. Raise your Pull Request
+### 4. Run tests locally
+Run the following command to run the registry's validation checks, for only the chain you added (replace the chain name or ID accordingly):
+```
+go test -run=/OP-Sepolia
```

Comment:
Since this is a regex, would `-run/OP` run both OP Mainnet and OP Sepolia? Either way, I'm good with this approach instead of #154—it seems simpler to maintain. The forward slash requirement is a bit quirky but not a dealbreaker

Reply:
You probably don't need the `/`.

Reply:
yes, doing `-run=OP` would run OP Mainnet & Sepolia

Reply:
The `/` changes the behaviour. Essentially there are two regexes, the one before the slash filters the test name, the one after filters the sub test name. Sub tests have been checked to ensure they are named per chain.

Compare
```
➜  validation git:(gk/focus-tests2) ✗ go test -run=OP -v                    
=== RUN   TestL2OOParams
=== RUN   TestL2OOParams/Orderly_Network_(291)
    utils_test.go:59: 
=== RUN   TestL2OOParams/OP-Sepolia_(11155420)
=== RUN   TestL2OOParams/Zora_Sepolia_(999999999)
    l2oo_test.go:86: 
=== RUN   TestL2OOParams/Base_devnet_0_(11763072)
=== RUN   TestL2OOParams/Base_(8453)
=== RUN   TestL2OOParams/PGN_(424)
    utils_test.go:59: 
=== RUN   TestL2OOParams/OP_Labs_Sepolia_devnet_0_(11155421)
=== RUN   TestL2OOParams/Mode_(34443)
=== RUN   TestL2OOParams/Zora_(7777777)
=== RUN   TestL2OOParams/PGN_Sepolia_(58008)
    utils_test.go:59: 
=== RUN   TestL2OOParams/Lyra_(957)
    utils_test.go:59: 
=== RUN   TestL2OOParams/OP-Mainnet_(10)
=== RUN   TestL2OOParams/Base_Sepolia_(84532)
--- PASS: TestL2OOParams (5.29s)
    --- SKIP: TestL2OOParams/Orderly_Network_(291) (0.00s)
    --- PASS: TestL2OOParams/OP-Sepolia_(11155420) (0.36s)
    --- SKIP: TestL2OOParams/Zora_Sepolia_(999999999) (0.00s)
    --- PASS: TestL2OOParams/Base_devnet_0_(11763072) (0.17s)
    --- PASS: TestL2OOParams/Base_(8453) (0.35s)
    --- SKIP: TestL2OOParams/PGN_(424) (0.00s)
    --- PASS: TestL2OOParams/OP_Labs_Sepolia_devnet_0_(11155421) (0.18s)
    --- PASS: TestL2OOParams/Mode_(34443) (0.18s)
    --- PASS: TestL2OOParams/Zora_(7777777) (3.68s)
    --- SKIP: TestL2OOParams/PGN_Sepolia_(58008) (0.00s)
    --- SKIP: TestL2OOParams/Lyra_(957) (0.00s)
    --- PASS: TestL2OOParams/OP-Mainnet_(10) (0.19s)
    --- PASS: TestL2OOParams/Base_Sepolia_(84532) (0.17s)
PASS
ok      github.com/ethereum-optimism/superchain-registry/validation     5.788s
```
with
```
➜  validation git:(gk/focus-tests2) ✗ go test -run=/OP -v
=== RUN   TestGasPriceOracleParams
=== RUN   TestGasPriceOracleParams/OP-Mainnet_(10)
=== RUN   TestGasPriceOracleParams/OP-Sepolia_(11155420)
--- PASS: TestGasPriceOracleParams (0.88s)
    --- PASS: TestGasPriceOracleParams/OP-Mainnet_(10) (0.47s)
    --- PASS: TestGasPriceOracleParams/OP-Sepolia_(11155420) (0.41s)
=== RUN   TestL2OOParams
=== RUN   TestL2OOParams/OP-Mainnet_(10)
=== RUN   TestL2OOParams/OP-Sepolia_(11155420)
=== RUN   TestL2OOParams/OP_Labs_Sepolia_devnet_0_(11155421)
--- PASS: TestL2OOParams (0.77s)
    --- PASS: TestL2OOParams/OP-Mainnet_(10) (0.29s)
    --- PASS: TestL2OOParams/OP-Sepolia_(11155420) (0.30s)
    --- PASS: TestL2OOParams/OP_Labs_Sepolia_devnet_0_(11155421) (0.18s)
=== RUN   TestResourceConfig
=== RUN   TestResourceConfig/OP-Mainnet_(10)
=== RUN   TestResourceConfig/OP-Sepolia_(11155420)
=== RUN   TestResourceConfig/OP_Labs_Sepolia_devnet_0_(11155421)
--- PASS: TestResourceConfig (0.14s)
    --- PASS: TestResourceConfig/OP-Mainnet_(10) (0.04s)
    --- PASS: TestResourceConfig/OP-Sepolia_(11155420) (0.05s)
    --- PASS: TestResourceConfig/OP_Labs_Sepolia_devnet_0_(11155421) (0.05s)
=== RUN   TestGenesisHash
=== RUN   TestGenesisHash/OP-Mainnet_(10)
    superchain-genesis_test.go:42: chain 10: EXCLUDED from Genesis block hash validation
=== RUN   TestGenesisHash/OP-Sepolia_(11155420)
=== RUN   TestGenesisHash/OP_Labs_Sepolia_devnet_0_(11155421)
--- PASS: TestGenesisHash (0.24s)
    --- SKIP: TestGenesisHash/OP-Mainnet_(10) (0.00s)
    --- PASS: TestGenesisHash/OP-Sepolia_(11155420) (0.12s)
    --- PASS: TestGenesisHash/OP_Labs_Sepolia_devnet_0_(11155421) (0.12s)
=== RUN   TestSuperchainWideContractVersions
--- PASS: TestSuperchainWideContractVersions (0.00s)
=== RUN   TestContractVersions
=== RUN   TestContractVersions/OP-Mainnet_(10)
=== RUN   TestContractVersions/OP-Sepolia_(11155420)
=== RUN   TestContractVersions/OP_Labs_Sepolia_devnet_0_(11155421)
    superchain-version_test.go:98: chain 11155421: EXCLUDED from contract version validation
--- PASS: TestContractVersions (1.86s)
    --- PASS: TestContractVersions/OP-Mainnet_(10) (0.29s)
    --- PASS: TestContractVersions/OP-Sepolia_(11155420) (1.57s)
    --- SKIP: TestContractVersions/OP_Labs_Sepolia_devnet_0_(11155421) (0.00s)
=== RUN   TestAreCloseInts
--- PASS: TestAreCloseInts (0.00s)
PASS
ok      github.com/ethereum-optimism/superchain-registry/validation     4.145s
```

Reply:
Oh I see, we have are getting a match on L2OOParams. Keep the `/`